recipes/amzn-sagemaker-aiops-jupyterlab-extension/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "amzn-sagemaker-aiops-jupyterlab-extension" %}
+{% set version = "1.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/amzn_sagemaker_aiops_jupyterlab_extension-{{ version }}.tar.gz
+  sha256: 0250bb2081c0a29425c169af03cc2096776aff7a62fc14c795b61c806a9e9cc1
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >={{ python_min }}
+    - hatchling >=1.5.0
+    - jupyterlab >=4.0,<5.dev0
+    - hatch-nodejs-version >=0.3.2
+    - hatch-jupyter-builder
+    - pip
+  run:
+    - python >={{ python_min }}
+    - jupyterlab >=4
+    - jupyter_events >=0.6.0
+    - async-lru
+    - aws-embedded-metrics
+    - aiobotocore >=2.7.0
+    - y-py >=0.6.0,<0.7.0
+    - ypy-websocket >=0.12.0
+    - nbformat >=5.9.2
+    - pydantic >=1.10.17,<3
+
+test:
+  commands:
+    - pip check
+    {% if not win %}
+    - jupyter labextension list 2>&1 | grep -q "@amzn/amzn_sagemaker_aiops_jupyterlab_extension"
+    {% else %}
+    - jupyter labextension list 2>&1 | findstr "@amzn/amzn_sagemaker_aiops_jupyterlab_extension"
+    {% endif %}
```

It is noarch, no need to make this test pass in Windows here b/c it will be build on Linux only in the feedstock.

```suggestion
    - jupyter labextension list 2>&1 | grep -q "@amzn/amzn_sagemaker_aiops_jupyterlab_extension"
```

---

Hi @ocefpaf , I had this earlier and build failed,  which is why I added this. https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1245977&view=logs&j=53c3d5f4-e7c7-5b53-4b94-211701b2ba17&t=8f042650-8479-5761-7a4a-8d021c0bb2de&l=2399

---

Please read mu comment above. It is okay to fail here on windows.

---

Ah, I see! Just pulled the changes!